pkg/kubelet/cm/helpers_linux.go

```diff
@@ -203,6 +206,41 @@ func GetCgroupSubsystems() (*CgroupSubsystems, error) {
 	}, nil
 }
 
+// getCgroupSubsystemsV2 returns information about the enabled cgroup v2 subsystems
+func getCgroupSubsystemsV2() (*CgroupSubsystems, error) {
+	content, err := ioutil.ReadFile(filepath.Join(util.CgroupRoot, "cgroup.controllers"))
+	if err != nil {
+		return nil, err
+	}
+
+	mounts := []libcontainercgroups.Mount{}
+	controllers := strings.Split(strings.Trim(string(content), "\n"), " ")
+	mountPoints := make(map[string]string, len(controllers))
+	for _, controller := range controllers {
+		mountPoints[controller] = util.CgroupRoot
+		m := libcontainercgroups.Mount{
+			Mountpoint: util.CgroupRoot,
+			Root:       util.CgroupRoot,
+			Subsystems: []string{controller},
+		}
+		mounts = append(mounts, m)
+	}
+
+	return &CgroupSubsystems{
```

Comment:
can we ditch "subsystem mountpoint" concept for v2 codes?

runc and containerd are trying to eliminate all "subsystem" concept: https://github.com/opencontainers/runc/pull/2169 https://godoc.org/github.com/containerd/cgroups/v2

Reply:
@giuseppe WDYT?

Reply:
is this refactoring something that could be addressed later?  I'd like to get the current version merged as soon as possible, so that we can already start using it

Reply:
SGTM